platform.js

```diff
@@ -6,12 +6,15 @@ os = require("os"),
 fs = require(process.versions.electron ? "original-fs" : "fs"),
 ncp = require("ncp"),
 rimraf = require("rimraf"),
+net = require("net"),
 gunzip = require("gunzip-maybe"),
 tar = require("tar-fs"),
 ws = require("windows-shortcuts"),
-tempDir = "rvplayer-" + new Date().getTime();
+tempDir = "rvplayer-" + new Date().getTime(),
+componentsInfo = {};
```

Comment:
@fjvallarino it seems we are never setting any properties on `componentsInfo`, or am I missing something?

Also, what is it supposed to do? Can we think of a more descriptive name? (something that implies the type of information it's supposed to hold)